modular_skyrat/modules/microfusion/code/microfusion_designs.dm

```diff
@@ -13,20 +13,28 @@
 	id = "enhanced_microfusion_phase_emitter"
 	materials = list(/datum/material/iron = 1000, /datum/material/glass = 1000, /datum/material/silver = 500)
 	build_path = /obj/item/microfusion_phase_emitter/enhanced
+	category = list(
+		RND_CATEGORY_INITIAL, RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_PARTS
+	)
 
 /datum/design/microfusion/advanced_phase_emitter
 	name = "Advanced Microfusion Phase Emitter"
 	desc = "The core of a microfusion projection weapon, produces the laser."
 	id = "advanced_microfusion_phase_emitter"
 	materials = list(/datum/material/iron = 1000, /datum/material/glass = 1000, /datum/material/silver = 500, /datum/material/gold = 500)
 	build_path = /obj/item/microfusion_phase_emitter/advanced
-
+	category = list(
+		RND_CATEGORY_INITIAL, RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_PARTS
+	)
```

Comment:
My brother you're going to have to go through these and ensure that you restore all of the newlines you've accidentally removed with what you've done, it's done that way in multiple spots in this file.

Reply:
all updated - that as my oversight. I did also add in the extra spacing between the category comments while i was there